### DISCUSSION TITLE
Difficulties in 3D flow field inversion

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#705] [General]
Hello, I currently need to perform flow field inversion on three-dimensional hills, but I only have high-precision data at the symmetry plane, so I choose to use 'varianxce' for it. If I use speed, how should I write UData? I have seen the example of "ramp" before, and the UData in it is full flow field data. However, I am currently lacking full flow field data. What should I do? Looking forward to your reply.

**Comment by friedenhe** [comment]
If your data is at the 2D sym plane only, how do you want the 3D case to match the data? Please elaborate more on your setup.
